src/common/PasswordInput.js

```diff
@@ -16,11 +16,19 @@ const styles = createStyleSheet({
     top: 0,
     bottom: 0,
     justifyContent: 'center',
+    borderTopWidth: 1,
+    borderColor: BORDER_COLOR,
+    borderBottomWidth: 1,
+    borderRightWidth: 1,
```

Comment:
I expect this to look odd on Android, where the inputs aren't styled with a border (see use of `Platform.select` in Input.js).